packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.tsx

```diff
@@ -0,0 +1,364 @@
+import {
+  Box,
+  CardMedia,
+  Link,
+  Stack,
+  Typography,
+  Skeleton,
+} from '@mui/material'
+import {
+  FileHandleAssociateType,
+  FileHandleAssociation,
+  QueryBundleRequest,
+  Row,
+} from '@sage-bionetworks/synapse-types'
+import useGetQueryResultBundle from '../../synapse-queries/entity/useGetQueryResultBundle'
+import { useGetStablePresignedUrl } from '../../synapse-queries'
+import { getFieldIndex } from '../../utils/functions/queryUtils'
+import * as SynapseConstants from '../../utils/SynapseConstants'
+import { parseEntityIdFromSqlStatement } from '../../utils/functions/SqlFunctions'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import dayjs from 'dayjs'
+
+export type FeaturedResearchProps = {
+  sql: string
+}
+
+export type FeaturedResearchCardProps = {
+  research: Row
+  entityId: string
+  isLoading: boolean
+  affiliationColIndex: number
+  publicationDateColIndex: number
+  titleColIndex: number
+  descriptionColIndex: number
+  tagsColIndex: number
+  linkColIndex: number
+  imageColIndex: number
+}
+
+const useImageUrl = (fileId: string, entityId: string) => {
+  const fha: FileHandleAssociation = {
+    associateObjectId: entityId,
+    associateObjectType: FileHandleAssociateType.TableEntity,
+    fileHandleId: fileId || '',
+  }
+  const stablePresignedUrl = useGetStablePresignedUrl(fha, false, {
+    enabled: !!fileId,
+  })
+  const dataUrl = stablePresignedUrl?.dataUrl
+
+  return dataUrl
+}
+
+const parseTags = (
+  tagsColIndex: number,
+  research: { values: (string | null)[] },
+): string[] => {
+  try {
+    const tags = (
+      research.values[tagsColIndex]
+        ? JSON.parse(research.values[tagsColIndex] || '')
+        : []
+    ) as string[]
+    return tags
+  } catch (e) {
+    console.error(e)
+    return []
+  }
+}
+
+const FeaturedResearchCard = ({
+  research,
+  entityId,
+  affiliationColIndex,
+  publicationDateColIndex,
+  titleColIndex,
+  tagsColIndex,
+  linkColIndex,
+  imageColIndex,
+  isLoading,
+}: FeaturedResearchCardProps) => {
+  const fileId = research.values[imageColIndex] ?? ''
+  const url = useImageUrl(fileId ?? '', entityId)
+  if (isLoading) {
+    return <Skeleton variant="rectangular" height={142} width="100%" />
+  }
+  return (
+    <Box
+      display={'flex'}
+      gap={'30px'}
+      borderBottom={'1px solid'}
+      padding={'24px 0'}
```

Comment:
FYI, I just learned these system props are deprecated in MUI v6, so you will want to get in the habit of putting these in `sx`. See https://mui.com/material-ui/migration/migrating-from-deprecated-apis/#system-props

Reply:
No need to change these in this PR, I can run a codemod to fix them in my future PR to update to MUI 6! But just an FYI